database/rrdhost.c

```diff
@@ -1303,6 +1303,8 @@ static void rrdhost_load_auto_labels(void) {
 
     add_aclk_host_labels();
 
+    health_add_host_labels();
```

Comment:
Aaargh, sorry for this, just a tiny bit, since we have `add_aclk_`, could we also have `add_health_`? Sorry, it's ocd, feel free to ignore it. :-)